### PR TITLE
Adding BR/MR

### PR DESCRIPTION
**Pull request by Anthogator**
Almost everything here is working just fine! I've only found two bugs I can't tackle:

The image for Coilgun BRs and MRs is not showing up in the equipment screen.

Laser and Coilgun Marksman Rifles are not currently giving the ability to shoot. I can't figure this one out. You can overwatch, but not take a shot. CV, MG, and BM all seem to work fine. All tiers of BR work fine.

I also made the changes I discovered after making my last PR (like stuff for the bullpup that got added in after I started working on it).

I also changed both Assault Rifle ammo to 3 and SMG ammo to 2, as per the balance requesting of JLPH, since we're adding BRs.

### CODE REVIEW

File: LWOTC_WeaponPack/Config/XComGameData_WeaponData.ini [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+-ASSAULTRIFLE_CONVENTIONAL_ICLIPSIZE=4
+-ASSAULTRIFLE_MAGNETIC_BASEDAMAGE=(Damage=6, Spread=1, PlusOne=0, Crit=3, Pierce=0, Shred=0, Tag="", DamageType="Projectile_MagXCom")
+-ASSAULTRIFLE_MAGNETIC_ICLIPSIZE=4
+-ASSAULTRIFLE_BEAM_BASEDAMAGE=(Damage=8, Spread=1, PlusOne=0, Crit=3, Pierce=0, Shred=0, Tag="", DamageType="Projectile_BeamXCom")
+-ASSAULTRIFLE_BEAM_ICLIPSIZE=4
++ASSAULTRIFLE_CONVENTIONAL_ICLIPSIZE=3
++ASSAULTRIFLE_MAGNETIC_BASEDAMAGE=(Damage=7, Spread=2, PlusOne=0, Crit=3, Pierce=0, Shred=0, Tag="", DamageType="Projectile_MagXCom")
++ASSAULTRIFLE_MAGNETIC_ICLIPSIZE=3
++ASSAULTRIFLE_BEAM_BASEDAMAGE=(Damage=8, Spread=1, PlusOne=0, Crit=3, Pierce=0, Shred=0, Tag="", DamageType="Projectile_BeamXCom")
++ASSAULTRIFLE_BEAM_ICLIPSIZE=3
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComGameData_WeaponData.ini]: Can you put this ammo change in a different PR? Otherwise this part is fine

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 2]
```diff
@@ -106,17 +130,41 @@ Bullpup_COIL_SCHEMATIC_ELERIUMCOST=5
...
+MarksmanRifle_BEAM_SCHEMATIC_ALLOYCOST = 20
+MarksmanRifle_BEAM_SCHEMATIC_ELERIUMCOST = 10
 ; TODO
 
 [LWOTC_WeaponPack.X2Item_LWConventionalWeapons]
 SMG_CONVENTIONAL_BASEDAMAGE=	(Damage=3, Spread=1, PlusOne=0, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Projectile_Conventional")
 SMG_CONVENTIONAL_AIM = 0
 SMG_CONVENTIONAL_CRITCHANCE = 0
-SMG_CONVENTIONAL_ICLIPSIZE = 3
+SMG_CONVENTIONAL_ICLIPSIZE = 2
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini]: Please put the ammo change in a separate PR

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 3]
```diff
@@ -136,37 +184,67 @@ MIDSHORT_CONVENTIONAL_RANGE[13] = -8
...
 
 [LWOTC_WeaponPack.X2Item_LaserWeapons]
 ; ***** ImagePaths *****
 AssaultRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserRifle_Base"
+BattleRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserRifle_Base"
 SMG_Laser_ImagePath="UILibrary_LW_LaserPack.LaserSMG_Base"
 Cannon_Laser_ImagePath="UILibrary_LW_LaserPack.LaserCannon_Base"
 Shotgun_Laser_ImagePath="UILibrary_LW_LaserPack.LaserShotgun_Base"
 SniperRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserSniper_Base"
+MarksmanRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserSniper_Base"
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini:216]: Did realitymachina's pack not have these assets? If not I'll need to put in an art request. Let me know

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 4]
```diff
@@ -136,37 +184,67 @@ MIDSHORT_CONVENTIONAL_RANGE[13] = -8
...
+MEDLONG_CONVENTIONAL_RANGE[13] = 0
+MEDLONG_CONVENTIONAL_RANGE[14] = 0
+MEDLONG_CONVENTIONAL_RANGE[15] = 0
+MEDLONG_CONVENTIONAL_RANGE[16] = 0
 ; TODO
 
 [LWOTC_WeaponPack.X2Item_LaserWeapons]
 ; ***** ImagePaths *****
 AssaultRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserRifle_Base"
+BattleRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserRifle_Base"
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini:211]: Did realitymachina's pack not have these assets? If not I'll need to put in an art request. Let me know

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 5]
```diff
@@ -136,37 +184,67 @@ MIDSHORT_CONVENTIONAL_RANGE[13] = -8
...
 SNIPERRIFLE_LASER_BASEDAMAGE=	(Damage=6, Spread=1, PlusOne=0, Crit=3, Pierce=0, Shred=0, Tag="", DamageType="Projectile_BeamXCom")
+MARKSMANRIFLE_LASER_BASEDAMAGE=	(Damage=5, Spread=1, PlusOne=0, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Projectile_BeamXCom")
 PISTOL_LASER_BASEDAMAGE=		(Damage=2, Spread=1, PlusOne=50, Crit=1, Pierce=0, Shred=0, Tag="", DamageType="Projectile_BeamXCom")
 BULLPUP_LASER_BASEDAMAGE=		(Damage=4, Spread=1, PlusOne=0, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Projectile_BeamXCom")
 
 ; Core properties and variables for XCom Laser Weapons
 ASSAULTRIFLE_LASER_AIM = 5
 ASSAULTRIFLE_LASER_CRITCHANCE = 0
-ASSAULTRIFLE_LASER_ICLIPSIZE = 4
+ASSAULTRIFLE_LASER_ICLIPSIZE = 3
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini]: Ammo change in new PR

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 6]
```diff
@@ -136,37 +184,67 @@ MIDSHORT_CONVENTIONAL_RANGE[13] = -8
...
+BATTLERIFLE_LASER_CRITCHANCE = 0
+BATTLERIFLE_LASER_ICLIPSIZE = 2
+BATTLERIFLE_LASER_ISOUNDRANGE = 27
+BATTLERIFLE_LASER_IENVIRONMENTDAMAGE = 10
+BATTLERIFLE_LASER_UPGRADESLOTS = 2
+
 SMG_LASER_AIM = 5
 SMG_LASER_CRITCHANCE = 0
-SMG_LASER_ICLIPSIZE = 3
+SMG_LASER_ICLIPSIZE = 2
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini]: Ammo change

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 7]
```diff
@@ -341,11 +462,27 @@ BULLPUP_LS_ELERIUMCOST=0
 SMG_MAGNETIC_BASEDAMAGE=(Damage=5, Spread=1, PlusOne=0, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Projectile_MagXCom")
 SMG_MAGNETIC_AIM = 0
 SMG_MAGNETIC_CRITCHANCE = 0
-SMG_MAGNETIC_ICLIPSIZE = 3
+SMG_MAGNETIC_ICLIPSIZE = 2
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini]: ammo change

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 8]
```diff
@@ -364,28 +501,55 @@ MIDSHORT_MAGNETIC_RANGE[13] = -8
...
 SNIPERRIFLE_COIL_BASEDAMAGE=			(Damage=8,	Spread=2, PlusOne=0,	Crit=4, Pierce=1, Shred=0, Tag="", DamageType="Projectile_MagXCom")
+MARKSMANRIFLE_COIL_BASEDAMAGE=			(Damage=7,	Spread=2, PlusOne=0,	Crit=3, Pierce=1, Shred=0, Tag="", DamageType="Projectile_MagXCom")
 SMG_COIL_BASEDAMAGE=					(Damage=6,	Spread=2, PlusOne=0,	Crit=3, Pierce=1, Shred=0, Tag="", DamageType="Projectile_MagXCom")
 PISTOL_COIL_BASEDAMAGE=					(Damage=3,	Spread=1, PlusOne=50,	Crit=1, Pierce=1, Shred=0, Tag="", DamageType="Projectile_MagXCom")
 BULLPUP_COIL_BASEDAMAGE=				(Damage=6,	Spread=2, PlusOne=0,	Crit=3, Pierce=1, Shred=0, Tag="", DamageType="Projectile_MagXCom")
 
 ASSAULTRIFLE_COIL_AIM=0
 ASSAULTRIFLE_COIL_CRITCHANCE=0
-ASSAULTRIFLE_COIL_ICLIPSIZE=4
+ASSAULTRIFLE_COIL_ICLIPSIZE=3
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini]: ammo change

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 9]
```diff
@@ -364,28 +501,55 @@ MIDSHORT_MAGNETIC_RANGE[13] = -8
...
+BATTLERIFLE_COIL_CRITCHANCE=0
+BATTLERIFLE_COIL_ICLIPSIZE=2
+BATTLERIFLE_COIL_ISOUNDRANGE=30
+BATTLERIFLE_COIL_IENVIRONMENTDAMAGE=10
+BATTLERIFLE_COIL_UPGRADESLOTS=2
+
 SMG_COIL_AIM=0
 SMG_COIL_CRITCHANCE=0
-SMG_COIL_ICLIPSIZE=3
+SMG_COIL_ICLIPSIZE=2
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini]: ammo change

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 10]
```diff
@@ -425,10 +596,12 @@ BULLPUP_COIL_IENVIRONMENTDAMAGE=5
 BULLPUP_COIL_UPGRADESLOTS=2
 
 AssaultRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilRifle_Base"
+BattleRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilRifle_Base"
 SMG_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilSMG_Base"
 Cannon_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilCannon_Base"
 Shotgun_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilShotgun_Base"
 SniperRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilSniperRifle_Base"
+MarksmanRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilSniperRifle_Base"
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini:604]: Did realitymachina's pack not have these assets? If not I'll need to put in an art request. Let me know

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 11]
```diff
@@ -425,10 +596,12 @@ BULLPUP_COIL_IENVIRONMENTDAMAGE=5
 BULLPUP_COIL_UPGRADESLOTS=2
 
 AssaultRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilRifle_Base"
+BattleRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilRifle_Base"
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini:599]: Did realitymachina's pack not have these assets? If not I'll need to put in an art request. Let me know

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 12]
```diff
@@ -562,11 +770,27 @@ BULLPUP_CG_ELERIUMCOST=2
 SMG_BEAM_BASEDAMAGE=			(Damage=7, Spread=1, PlusOne=0, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Projectile_BeamXCom")
 SMG_BEAM_AIM = 0
 SMG_BEAM_CRITCHANCE = 0
-SMG_BEAM_ICLIPSIZE = 3
+SMG_BEAM_ICLIPSIZE = 2
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini]: ammo

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc [hunk 13]
```diff
@@ -118,19 +156,29 @@ static function AddSchematicLoc (X2ItemTemplateManager ItemTemplateMgr)
...
 
 	CopySchematicLoc (ItemTemplateMgr, 'Pistol_LS_Schematic', 'Pistol_MG_Schematic');
 	CopySchematicLoc (ItemTemplateMgr, 'Pistol_CG_Schematic', 'Pistol_MG_Schematic');
-		
-	CopySchematicLoc (ItemTemplateMgr, 'Bullpup_LS_Schematic', 'Bullpup_MG_Schematic');
-	CopySchematicLoc (ItemTemplateMgr, 'Bullpup_CG_Schematic', 'Bullpup_MG_Schematic');
 
+	CopySchematicLoc (ItemTemplateMgr, 'Bullpup_LS_Schematic', 'Bullpup_MG_Schematic');
+ 	CopySchematicLoc (ItemTemplateMgr, 'Bullpup_CG_Schematic', 'Bullpup_MG_Schematic');
+
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc]: What changed here? Whitespace?

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc [hunk 14]
```diff
@@ -396,7 +444,6 @@ static function AddLaserCritUpgrade(X2ItemTemplateManager ItemTemplateManager, N
 
 	// Cannon
 	Template.AddUpgradeAttachment('Optic', 'UIPawnLocation_WeaponUpgrade_Cannon_Optic', "LWCannon_LS.Meshes.SK_LaserCannon_Optic_B", "", 'Cannon_LS', , "img:///UILibrary_LW_LaserPack.LaserCannon_OpticB", "img:///UILibrary_LW_LaserPack.Inv_Laser_OpticC", "img:///UILibrary_StrategyImages.X2InventoryIcons.Inv_weaponIcon_scope");
-
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc [hunk 15]
```diff
@@ -427,7 +474,7 @@ static function AddLaserAimBonusUpgrade(X2ItemTemplateManager ItemTemplateManage
 
 	// Cannon
 	Template.AddUpgradeAttachment('Optic', 'UIPawnLocation_WeaponUpgrade_Cannon_Optic', "LWCannon_LS.Meshes.SK_LaserCannon_Optic_A", "", 'Cannon_LS', , "img:///UILibrary_LW_LaserPack.LaserCannon_OpticA", "img:///UILibrary_LW_LaserPack.Inv_LaserCannon_OpticA", "img:///UILibrary_StrategyImages.X2InventoryIcons.Inv_weaponIcon_scope");
-
+
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc [hunk 16]
```diff
@@ -738,7 +784,7 @@ static function UpdateLaserWeaponTemplates()
 		UpdateWeaponTemplate(ItemTemplateManager, 'Cannon_MG', 'Cannon_LS');
 
 		UpdateWeaponTemplate(ItemTemplateManager, 'Pistol_MG', 'Pistol_LS');
-
+
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc [hunk 17]
```diff
@@ -757,7 +803,7 @@ static function UpdateLaserWeaponTemplates()
 		UpdateWeaponTemplate(ItemTemplateManager, 'Cannon_BM', 'Cannon_CG');
 
 		UpdateWeaponTemplate(ItemTemplateManager, 'Pistol_BM', 'Pistol_CG');
-
+
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc [hunk 18]
```diff
@@ -845,6 +891,7 @@ static function UpdateCoilAttachmentTemplates()
 	AddCoilFreeKillUpgrade(ItemTemplateManager, 'FreeKillUpgrade_Bsc');
 	AddCoilFreeKillUpgrade(ItemTemplateManager, 'FreeKillUpgrade_Adv');
 	AddCoilFreeKillUpgrade(ItemTemplateManager, 'FreeKillUpgrade_Sup');
+
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2DownloadableContentInfo_LWOTCWeaponPack.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 19]
```diff
@@ -159,11 +186,11 @@ static function X2DataTemplate CreateAssaultRifle_Coil_Template()
 	if (class'X2Item_CoilSchematics'.default.USE_SCHEMATICS)
 	{
 		Template.CreatorTemplateName = 'AssaultRifle_CG_Schematic'; // The schematic which creates this item
-		Template.BaseItem = 'AssaultRifle_MG'; // Which item this will be upgraded from
+		Template.BaseItem = 'AssaultRifle_MG'; // Which item this will be upgraded from
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 20]
```diff
@@ -99,13 +124,15 @@ var config int BULLPUP_CG_ELERIUMCOST;
 
 static function array<X2DataTemplate> CreateTemplates()
 {
-	local array<X2DataTemplate> Weapons;
+    local array<X2DataTemplate> Weapons;
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 21]
```diff
@@ -159,11 +186,11 @@ static function X2DataTemplate CreateAssaultRifle_Coil_Template()
 	if (class'X2Item_CoilSchematics'.default.USE_SCHEMATICS)
 	{
 		Template.CreatorTemplateName = 'AssaultRifle_CG_Schematic'; // The schematic which creates this item
-		Template.BaseItem = 'AssaultRifle_MG'; // Which item this will be upgraded from
+		Template.BaseItem = 'AssaultRifle_MG'; // Which item this will be upgraded from		
 	}
 	else
 	{
-		Template.Requirements.RequiredTechs.AddItem(class'X2StrategyElement_CoilTechs'.default.CoilWeaponTech_Tier[0]);
+		Template.Requirements.RequiredTechs.AddItem(class'X2StrategyElement_CoilTechs'.default.CoilWeaponTech_Tier[0]);
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 22]
```diff
@@ -458,11 +558,11 @@ static function X2DataTemplate CreateSniperRifle_Coil_Template()
 	if (class'X2Item_CoilSchematics'.default.USE_SCHEMATICS)
 	{
 		Template.CreatorTemplateName = 'SniperRifle_CG_Schematic'; // The schematic which creates this item
-		Template.BaseItem = 'SniperRifle_MG'; // Which item this will be upgraded from
+		Template.BaseItem = 'SniperRifle_MG'; // Which item this will be upgraded from		
 	}
 	else
 	{
-		Template.Requirements.RequiredTechs.AddItem(class'X2StrategyElement_CoilTechs'.default.CoilWeaponTech_Tier[1]);
+		Template.Requirements.RequiredTechs.AddItem(class'X2StrategyElement_CoilTechs'.default.CoilWeaponTech_Tier[1]);
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 23]
```diff
@@ -458,11 +558,11 @@ static function X2DataTemplate CreateSniperRifle_Coil_Template()
 	if (class'X2Item_CoilSchematics'.default.USE_SCHEMATICS)
 	{
 		Template.CreatorTemplateName = 'SniperRifle_CG_Schematic'; // The schematic which creates this item
-		Template.BaseItem = 'SniperRifle_MG'; // Which item this will be upgraded from
+		Template.BaseItem = 'SniperRifle_MG'; // Which item this will be upgraded from
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 24]
```diff
@@ -382,11 +482,11 @@ static function X2DataTemplate CreateShotgun_Coil_Template()
 	if (class'X2Item_CoilSchematics'.default.USE_SCHEMATICS)
 	{
 		Template.CreatorTemplateName = 'Shotgun_CG_Schematic'; // The schematic which creates this item
-		Template.BaseItem = 'Shotgun_MG'; // Which item this will be upgraded from
+		Template.BaseItem = 'Shotgun_MG'; // Which item this will be upgraded from		
 	}
 	else
 	{
-		Template.Requirements.RequiredTechs.AddItem(class'X2StrategyElement_CoilTechs'.default.CoilWeaponTech_Tier[1]);
+		Template.Requirements.RequiredTechs.AddItem(class'X2StrategyElement_CoilTechs'.default.CoilWeaponTech_Tier[1]);
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 25]
```diff
@@ -382,11 +482,11 @@ static function X2DataTemplate CreateShotgun_Coil_Template()
 	if (class'X2Item_CoilSchematics'.default.USE_SCHEMATICS)
 	{
 		Template.CreatorTemplateName = 'Shotgun_CG_Schematic'; // The schematic which creates this item
-		Template.BaseItem = 'Shotgun_MG'; // Which item this will be upgraded from
+		Template.BaseItem = 'Shotgun_MG'; // Which item this will be upgraded from
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 26]
```diff
@@ -309,11 +409,11 @@ static function X2DataTemplate CreateCannon_Coil_Template()
 	if (class'X2Item_CoilSchematics'.default.USE_SCHEMATICS)
 	{
 		Template.CreatorTemplateName = 'Cannon_CG_Schematic'; // The schematic which creates this item
-		Template.BaseItem = 'Cannon_MG'; // Which item this will be upgraded from
+		Template.BaseItem = 'Cannon_MG'; // Which item this will be upgraded from		
 	}
 	else
 	{
-		Template.Requirements.RequiredTechs.AddItem(class'X2StrategyElement_CoilTechs'.default.CoilWeaponTech_Tier[1]);
+		Template.Requirements.RequiredTechs.AddItem(class'X2StrategyElement_CoilTechs'.default.CoilWeaponTech_Tier[1]);
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 27]
```diff
@@ -309,11 +409,11 @@ static function X2DataTemplate CreateCannon_Coil_Template()
 	if (class'X2Item_CoilSchematics'.default.USE_SCHEMATICS)
 	{
 		Template.CreatorTemplateName = 'Cannon_CG_Schematic'; // The schematic which creates this item
-		Template.BaseItem = 'Cannon_MG'; // Which item this will be upgraded from
+		Template.BaseItem = 'Cannon_MG'; // Which item this will be upgraded from
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 28]
```diff
@@ -234,11 +334,11 @@ static function X2DataTemplate CreateSMG_Coil_Template()
 	if (class'X2Item_CoilSchematics'.default.USE_SCHEMATICS)
 	{
 		Template.CreatorTemplateName = 'SMG_CG_Schematic'; // The schematic which creates this item
-		Template.BaseItem = 'SMG_MG'; // Which item this will be upgraded from
+		Template.BaseItem = 'SMG_MG'; // Which item this will be upgraded from		
 	}
 	else
 	{
-		Template.Requirements.RequiredTechs.AddItem(class'X2StrategyElement_CoilTechs'.default.CoilWeaponTech_Tier[0]);
+		Template.Requirements.RequiredTechs.AddItem(class'X2StrategyElement_CoilTechs'.default.CoilWeaponTech_Tier[0]);
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 29]
```diff
@@ -234,11 +334,11 @@ static function X2DataTemplate CreateSMG_Coil_Template()
 	if (class'X2Item_CoilSchematics'.default.USE_SCHEMATICS)
 	{
 		Template.CreatorTemplateName = 'SMG_CG_Schematic'; // The schematic which creates this item
-		Template.BaseItem = 'SMG_MG'; // Which item this will be upgraded from
+		Template.BaseItem = 'SMG_MG'; // Which item this will be upgraded from
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc]: whitespace

File: LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc [hunk 30]
```diff
@@ -638,4 +812,4 @@ static function X2DataTemplate CreateBullpup_Coil_Template()
 	Template.DamageTypeTemplateName = 'Projectile_MagXCom';
 
 	return Template;
-}
```
Comment (TNychka) [inline LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_CoilWeapons.uc:641]: whitespace, mostly line endings on this entire file

**Review by TNychka — CHANGES_REQUESTED** [review]
Looks good, just a little bit of whitespace clean up now :)

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini (hunk 4 above)
Comment (Anthogator) [reply LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini:211]: Correct. RM's pack reused both the 2D assets for images of the BR and MR as well as the based model for 3D assets. The only difference in 3D models is that the BR and MR use some different weapon parts (the MRs [except LS and CG] have scopes, for example). LS and CG MRs look identical to LS and CG assault rifles right now.

**Comment by Anthogator** [comment]
All changes made, whitespace removed, mister boss

**Review by TNychka — APPROVED** [review]
Approved pending testing
